codeflash/optimization/function_optimizer.py

```diff
@@ -301,9 +304,30 @@ def optimize_function(self) -> Result[BestOptimization, str]:
                     code_context=code_context, optimized_code=best_optimization.candidate.source_code
                 )
 
-                new_code, new_helper_code = self.reformat_code_and_helpers(
-                    code_context.helper_functions, explanation.file_path, self.function_to_optimize_source_code
-                )
+                if not self.args.disable_imports_sorting:
```

Comment:
why are you doing this inline and not as a part of a new function?

Reply:
a new function will also make it easier to unit test

Reply:
Done.